Documentation/scenario-test-guide.md

```diff
@@ -0,0 +1,51 @@
+Scenario Testing Guide
+======================
+
+Overview
+--------
+WCF scenario tests are also referred to as outerloop tests. These are tests involving communication between WCF client tests and WCF test services. The WCF client tests running on .NET Core support cross platform, which means they can run on all supported platforms such as Windows, various Linux distros, Mac OS X as well as Universal Windows Platform (UWP). The WCF test services are built on full .NET Framwork that is supported on Windows only.
+
+Regardless which platform where you run WCF client tests, you will need a WCF test service for scenario tests to work. WCF Test services can be hosted in two ways like WCF users would usually do, ie., self-hosted and IIS-hosted. The following sections provide instructions of how to host WCF test services and how to run WCF scenario tests.
```

Comment:
Recommend ending the sentence "..., self-hosted or IIS-hosted on a Windows machine using the full .NET Framework (for example Windows Server 2012 R2 or Windows10 Enterprise)."